### PR TITLE
Feat/working links

### PR DESCRIPTION
**Pull request by checkeredpotato**
## What?
- Changed cta buttons into html link elements
- Changed margin for logo
- Changed margin for hero image

## Why?
To change buttons into links
To align logo close to where hero message starts
To visually make good use of white space and get align to the right side more

## How?
By using p and /p into a and /a
added 2.5 ems horizontally
added 8 ems horizontally

### CODE REVIEW

File: styles.less [hunk 1]
```diff
@@ -62,13 +62,15 @@ header h1 {
 }
 
 #section1-whitebox {
-  display: flex;
+  //display: flex;
```
Comment (heinrich10) [inline styles.less]: delete comments

File: styles.less [hunk 2]
```diff
@@ -89,18 +91,21 @@ header h1 {
 #section1-buttons-container {
   display: flex;
   gap: 10px;
-  height: 100px;
+  //height: 40px;
+  //border-radius: 8px;
```
Comment (heinrich10) [inline styles.less]: delete comments

File: styles.less [hunk 3]
```diff
@@ -89,18 +91,21 @@ header h1 {
...
   .fdirection()
+
 }
 
-#section1-buttons-container button {
+#section1-buttons-container a {
+  //border-radius: 8px;
```
Comment (heinrich10) [inline styles.less]: delete comments

File: styles.less [hunk 4]
```diff
@@ -196,15 +202,47 @@ header h1 {
...
-  padding: 20px;
-  @media screen and (min-width: 820px) {
-    display: flex;
+#section3-whitebox-right{
+  .fdirection();
+
+  display: flex;
+  align-items: center;
+  padding-bottom: 1em;
+  @media screen and (min-width: 820px)
```
Comment (heinrich10) [inline styles.less]: put { on line 211, follow the others

File: styles.less [hunk 5]
```diff
@@ -196,15 +202,47 @@ header h1 {
...
+    text-align: center;
+    margin: auto;
+    }
+
+
+}
+
+
+#section1-buttons-container a {
+  //border-radius: 8px;
```
Comment (heinrich10) [inline styles.less]: comments

File: styles.less [hunk 6]
```diff
@@ -256,13 +294,21 @@ footer {
...
-  padding: 0.3em 1.2em;
+  align-items: center;
+  height: 20px;
+  padding: 1em 1em;
   border-radius: 8px;
-  flex-shrink: 0;
+  //flex-shrink: 0;
   font-weight: 700;
+  text-decoration: none;
+  //color: #333333;
```
Comment (heinrich10) [inline styles.less]: comments

File: styles.less (hunk 1 above)
Comment (checkeredpotato) [reply styles.less]: deleted comment

File: styles.less (hunk 2 above)
Comment (checkeredpotato) [reply styles.less]: deleted comment

File: styles.less (hunk 3 above)
Comment (checkeredpotato) [reply styles.less]: deleted comment

File: styles.less (hunk 5 above)
Comment (checkeredpotato) [reply styles.less]: deleted comment

File: styles.less (hunk 6 above)
Comment (checkeredpotato) [reply styles.less]: deleted comment

**Review by heinrich10 — APPROVED** [review]

**Pull request merged**